state/rds-postgres.yaml

```diff
@@ -133,8 +133,8 @@ Parameters:
   EngineVersion:
     Description: 'PostgreSQL version.'
     Type: String
-    Default: '9.6.5'
-    AllowedValues: ['9.5.2', '9.5.4', '9.5.6', '9.5.7', '9.5.9', '9.5.10', '9.5.12', '9.5.13', '9.5.14', '9.5.15', '9.5.16', '9.5.18', '9.5.19', '9.5.20', '9.5.21', '9.5.22', '9.5.23', '9.6.1', '9.6.2', '9.6.3', '9.6.5', '9.6.6', '9.6.8', '9.6.9', '9.6.10', '9.6.11', '9.6.12', '9.6.14', '9.6.15', '9.6.16', '9.6.17', '9.6.18', '9.6.19', '10.1', '10.3', '10.4', '10.5', '10.6', '10.7', '10.9', '10.10', '10.11', '10.12', '10.13', '10.14', '11.1', '11.2', '11.4', '11.5', '11.6', '11.7', '11.8', '11.9', '12.2', '12.3', '12.4', '13.2'] # aws rds describe-db-engine-versions --engine postgres --query "DBEngineVersions[].EngineVersion"
+    Default: '9.6.9'
```

Comment:
@michaelwittig Why do we default to the oldest version? For backwards compatibility?

Reply:
Yes. Is 9.6.5 no longr available? I wonder if we should remove the default altogether.

Reply:
Exactly, `9.6.5` is no longer available.

Reply:
I see.